src/main/java/persistence/sql/ddl/EntityIdField.java

```diff
@@ -0,0 +1,34 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+
+import java.lang.reflect.Field;
+
+public record EntityIdField(EntityField field, GenerationType generationType) {
+    public static EntityIdField from(Field field) {
+        GeneratedValue generatedValue = field.getAnnotation(GeneratedValue.class);
+        GenerationType generationType = GenerationType.AUTO;
+
+        if (generatedValue != null) {
+            generationType = generatedValue.strategy();
+        }
```

Comment:
```suggestion
        GenerationType generationType = GenerationType.AUTO;

        if (generatedValue != null) {
            return new EntityIdField(
                    EntityField.from(field),
                    GenerationType.AUTO
        };
```
1. 가변적으로 변수를 사용하는 것보다는 바로 생성하여 리턴하도록 해도 되지 않을까요?
2.  `GeneratedValue.strategy()`의 기본값을 가져오는 목적이라면 아래와 같이 사용하여 쓸 수도 있습니다 :)
```java
GeneratedValue.class.getMethod("strategy")
    .getDefaultValue();
```